Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM node:16.13.0-slim as base
+FROM node:16.13.0-alpine as base
+WORKDIR /opt/graphistry-js
+COPY lerna.json package.json package-lock.json ./
+RUN apk add --no-cache python3 make g++
```

Comment:
why do we need need py3/g++/etc for pure node apps?

Reply:
explained in PR comment